### DISCUSSION TITLE
json key or value as an item title in app/collections

### DISCUSSION THREAD

**Discussion by emrahnazif** [directus/directus#15457] [Feature Requests]
In a collection with cards layout, I can select a json field as title or subtitle. 
I wish I could select a json key or value as a title or subtitle. 

<img width="273" alt="Screenshot 2022-09-07 at 16 13 56" src="https://user-images.githubusercontent.com/33692017/188900684-0f55fb51-edd5-4ee9-9d9a-2d9b4fde4db1.png">

**Comment by rijkvanzanten** [comment]
This directly relates to https://github.com/directus/directus/discussions/7277 🙂

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
